osgi/karaf-features/src/main/features/features.xml

```diff
@@ -17,10 +17,23 @@
         <bundle>mvn:org.jboss.logging/jboss-logging/${jboss.logging.version}</bundle>
         <bundle>mvn:com.fasterxml/classmate/${classmate.version}</bundle>
         <bundle>mvn:org.glassfish/javax.el/${javax.el.version}</bundle>
+    </feature>
+    <feature name="hibernate-validator-jsoup" version="${project.version}">
+        <feature>hibernate-validator</feature>
         <bundle>mvn:org.jsoup/jsoup/${jsoup.version}</bundle>
+    </feature>
+    <feature name="hibernate-validator-joda-time" version="${project.version}">
+        <feature>hibernate-validator</feature>
         <bundle>mvn:joda-time/joda-time/${joda-time.version}</bundle>
+    </feature>
+    <feature name="hibernate-validator-javax-money" version="${project.version}" description="Be aware that the JavaMoney API license is currently non-free">
```

Comment:
I don't think we need this comment there. Not feeling like having a discussion of what's free and what's not in a feature description.

Reply:
As you wish. I personally think it's important to raise awareness of that but I can remove it for sure.

Reply:
@gunnarmorling force-pushed an update removing the description.